config.js

```diff
@@ -113,39 +72,37 @@ exports.updateFromArgv = function(argv) {
 
     // 2) specific ranked/unranked options exports
 
-    processRankExport("minrank", argv);
-    processRankExport("minrankranked", argv);
-    processRankExport("minrankunranked", argv);
+    exportMinMaxRanks(argv);
+    exportBansUnderscored(argv);
+    exportAllowedGroupsUnderscored(argv);
```

Comment:
Why the need to mention underscored on these 2 funcitons?

Reply:
indeed this is redundant
i guess the first name exportInitialConfigUnderscored was really nice and i got a bit carried away :sweat_smile: